src/electron/index.ts

```diff
@@ -12,7 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import * as sentry from '@sentry/electron';
+// Directly import @sentry/electron main process code.
+// See: https://docs.sentry.io/platforms/javascript/guides/electron/#webpack-configuration
+import * as sentry from '@sentry/electron/dist/main';
```

Comment:
That doc is really helpful.

I think we should also fix the way we import sentry in the renderer process. By importing 
`@sentry/electron/dist/renderer` instead of `@sentry/electron`, we can probably get rid of the workaround we have here:
https://github.com/Jigsaw-Code/outline-client/blob/master/src/www/electron.webpack.js#L54-L57

This looks good though. Lmk if you rather us do that in a separate PR.

Reply:
I think that's a good idea.  Better to be more explicit anyways.

Reply:
That's a good idea, let's tackle it in a separate PR so we can release today.